Update: Alerts & Notifications (#2306)
getsentry/sentry-docs: src/docs/product/alerts-notifications/notifications/index.mdx

  - /learn/notifications/
---

> Sentry users with owner permissions and above can create notifications, change default notification settings, and configure notifications according to a project's needs. Notifications are categorized as [Workflow](#workflow), [Deploy](#deploy), and [Quota](#quota).

Author: @adhiraj I think this is the correct permissions for notifications. Will you confirm?

An issue can be in one of the following three states:

| Issue State       | Description                        |
| ----------------- | ---------------------------------- |
| Unresolved | The default state when an issue is added to the system. |

> | Resolved | Resolve an issue to indicate that no more events are expected. If another event is seen in the issue, the issue will revert to unresolved. This is called a **regression**. An issue is marked as resolved in one of three ways: 1. When an individual resolves it by manually changing the state on the issue page UI 2. Submitting a fix with the associated issue ID 3. When the project’s auto-resolve feature is configured. Auto-resolve is located in **Project Settings > General Settings**. For more details about submitting a fix, see documentation on [resolving issues](/product/releases/#after-associating-commits). |

Reviewer: ```suggestion
| Resolved | Resolve an issue to indicate that no more events are expected. If another event is seen in the issue, the issue will revert to unresolved, which often identifies a regression. An issue is marked as resolved in one of three ways: 1. When an individual resolves it by manually changing the state on the issue page UI 2. Submitting a fix with the associated issue ID 3. When the project’s auto-resolve feature is configured. Auto-resolve is located in **Project Settings > General Settings**. For more details about submitting a fix, see documentation on [resolving issues](/product/releases/#after-associating-commits). |
```
